diesel_cli/src/print_schema.rs

```diff
@@ -248,23 +266,66 @@ pub fn output_schema(
     Ok(out)
 }
 
+#[derive(Debug, PartialEq, Clone, Eq)]
+struct CustomColumnType {
+    table_name: TableName,
+    column_name: String,
+    column_type: ColumnType,
+}
+
+impl CustomColumnType {
+    fn new(
+        backend: Backend,
+        table_name: TableName,
+        column_name: String,
+        column_type: ColumnType,
+    ) -> Self {
+        #[cfg(feature = "mysql")]
+        if backend == Backend::Mysql {
```

Comment:
This code will be only included in the final binary if the `mysql` feature flag is enabled. That's mostly to remove unneeded code if the feature flag is disabled.
It will be only executed if the current database backend (== that database the current operation is executed against) is `Backend::Mysql`, because that code is only relevant for this case.

Reply:
Thats really cool, thanks!